apis/beacon/blobs/blobs.yaml

```diff
@@ -0,0 +1,69 @@
+get:
+  operationId: getBlobs
+  summary: Get blobs
+  description: |
+    Retrieves blobs for a given block id.
+    Depending on `Accept` header it can be returned either as json or as bytes serialized by SSZ.
+
+    If the `indices` parameter is specified, only the blobs with the specified indices will be returned. There are no guarantees
+    for the returned blobs in terms of ordering.
+
+    After the Fulu fork, only supernodes (which custody all data columns) are required to return blobs. Clients may implement
+    blob reconstruction logic for non-super nodes.
+  tags:
+    - Beacon
+  parameters:
+    - name: block_id
+      in: path
+      required: true
+      $ref: "../../../beacon-node-oapi.yaml#/components/parameters/BlockId"
+    - name: indices
+      in: query
+      description: Array of indices for blobs to request for in the specified block. Returns all blobs in the block if not specified.
+      required: false
+      schema:
+        type: array
+        uniqueItems: true
+        items:
+          $ref: "../../../beacon-node-oapi.yaml#/components/schemas/Uint64"
+  responses:
+    "200":
+      description: "Successful response"
+      content:
+        application/json:
+          schema:
+            title: GetBlobsResponse
+            type: object
+            required: [execution_optimistic, finalized, data]
+            properties:
+              execution_optimistic:
+                $ref: "../../../beacon-node-oapi.yaml#/components/schemas/ExecutionOptimistic"
+              finalized:
+                $ref: "../../../beacon-node-oapi.yaml#/components/schemas/Finalized"
+              data:
+                $ref: "../../../beacon-node-oapi.yaml#/components/schemas/Blobs"
```

Comment:
I think we have Blob but not Blobs

Reply:
yes but on the components/schemas in beacon-node-oapi.yaml I don't think that definition shows up unless i just can't find it, i thought it would be under the Blob definition

Reply:
I added it in this PR [here](https://github.com/ethereum/beacon-APIs/pull/546/files#diff-962ea34dc6ec0e65d4e1d6c8682e97c547e6e0d2f1168925210ff7ee47ef6ce0R409-R410)

Reply:
oops thanks